src/Sylius/Bundle/CoreBundle/Command/ShowAvailablePluginsCommand.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Command;
+
+use Sylius\Bundle\CoreBundle\Installer\Renderer\TableRenderer;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class ShowAvailablePluginsCommand extends Command
+{
+    protected function configure(): void
+    {
+        $this->setName('sylius:show-available-plugins');
+        $this->setDescription('Shows official Sylius Plugins');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): void
+    {
+        $output->writeln('<comment>Available official Sylius Plugins:</comment>');
+
+        $pluginTable = new TableRenderer($output);
+
+        $pluginTable->setHeaders(['Plugin', 'Description', 'URL']);
+        $pluginTable->addRow(['<info>Admin Order Creation</info>', 'Creating (and copying) orders in the administration panel.', 'https://github.com/Sylius/AdminOrderCreationPlugin']);
```

Comment:
IMHO a list of plugins with whole data as an array and for loop with this function would be nicer solution. Even better if we add VO :)

Reply:
VO `PluginInfo(string $name, string $description, string $url)` would be great 👌 but an array of data would also work well :)